guacamole/src/main/frontend/src/translations/ca.json

```diff
@@ -50,7 +58,10 @@
     "CLIENT" : {
 
         "ACTION_ACKNOWLEDGE"               : "@:APP.ACTION_ACKNOWLEDGE",
-        "ACTION_CLEAR_COMPLETED_TRANSFERS" : "Esborra",
+        "ACTION_CANCEL"                    : "@:APP.ACTION_CANCEL",
+        "ACTION_CLEAR_CLIENT_MESSAGES"     : "@:APP.ACTION_CLEAR",
+        "ACTION_CLEAR_COMPLETED_TRANSFERS" : "@:APP.ACTION_CLEAR",
+        "ACTION_CONTINUE"                  : "@:APP.ACTION_CONTINUE",
```

Comment:
I don't think you need to specifically include this ones that reference other translations.

Reply:
I just left them because i've seen other languages also have them. I thought that maybe without it, it would default to english.

Reply:
So, you think i can remove them all @necouchman ?

Reply:
Removed all the @APP lines